src/chat/ChatProvider.tsx

```diff
@@ -19,17 +19,23 @@ export const ChatProvider: React.FC<ChatProviderProps> = ({
   enableEncrypt = false,
   encryptKey = '',
   blackListWords = null,
+  encryptionOptions = null,
+  encryptionFuncProps = null,
```

Comment:
```suggestion
  encryptionFuncProps,
```
Same above, just let it be `undefined`

Reply:
because encryptionOptions used ChatProvider.tsx. If we not passed null in dependency will make useEffect loop forever. So I think should be put null value at here

Reply:
Same above

Reply:
removed it. Please check again